README.md

```diff
@@ -34,10 +34,13 @@ generated file by hand.
 
 ## Usage
 
+In a Unix or Node.js command prompt, use:
 ```
 node abp2blocklist.js < easylist.txt > easylist.json
```

Comment:
Please can you modify this example to use the names `input.txt` and `output.json` instead?